README.md

```diff
@@ -14,21 +14,98 @@ It is work in progress, but functions well.
 | ![](before.png)                                |                                 ![](after.png) |
 
 # Supported Languages
-- [x] [C][tree-sitter-c]
-- [x] [C++][tree-sitter-cpp]
-- [ ] CSS
-- [x] [D][tree-sitter-d]
-- [x] [Diff][tree-sitter-diff]
-- [x] [Go][tree-sitter-go]
-- [x] [go.mod][tree-sitter-go-mod]
-- [ ] HTML
-- [x] [Javascript/JSX][tree-sitter-javascript]
-- [x] [Julia][tree-sitter-julia]
-- [x] [Lua][tree-sitter-lua]
-- [x] [Rust][tree-sitter-rust]
-- [x] [Zig][tree-sitter-zig]
-
-If you want more languages supported, open an issue.
+
+Any language supported by `tree-sitter` should (in theory) be supported as well.
+
+It is possible to use grammar installed locally or directly from their git repository. Additionally 
+it is possible to specify where to find the queries of the language as well as use sub path for projects
+containing more then one language.
+## Default grammars
+
+A few grammars have default configurations, pre-compiled parsers and custom queries:
+
+The supported languages are:
+ 
```

Comment:
this can be formatted a bit better:

```suggestion
## Default grammars

A few grammars have default configurations, pre-compiled parsers and custom queries.
The supported languages are:  
```